src/components/Button.module.css

```diff
@@ -0,0 +1,9 @@
+.Button {
```

Comment:
The button should have borders ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/ButtonsBoarder.png)

Reply:
I tried to add the border on each button but it removed the clicking effect of buttons. Therefore, I left as it is and added the outline when a button is on focus.